docs/wiki/guides/current/VTX-CLI-Settings.md

```diff
@@ -4,7 +4,7 @@ As of Betaflight version 3.3.0, the CLI settings below can be used to
 configure addressable video transmitters (such as
 TBS-[SmartAudio](/docs/wiki/guides/current/SmartAudio)
 and
-IRC-[Tramp](IRC-Tramp)[)](IRC-Tramp)
+IRC-[Tramp](IRC-Tramp))
```

Comment:
```suggestion
IRC-[Tramp](IRC-Tramp)
```

Reply:
The bracket you're sugesting to remove is paired with the one on line 4

> configure addressable video transmitters **(**such as